dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryConfigServerVarTest.java

```diff
@@ -60,8 +60,24 @@ public static void setUp() throws Exception {
         server.startServer();
     }
 
+    @Test
+    public void testConfig() throws Exception {
+        runTest(server, APP_NAME + "/testConfig", "testServiceNameConfig");
+        runTest(server, APP_NAME + "/testConfig", "testSDKDisabledConfig");
+    }
```

Comment:
Why was this changed?
I assume it replaces the `@TestServlet` annotation, but I'm not sure why it was necessary to do so.

Reply:
I moved this test from using `@TestServlet` to using `runTest` because `testWarningForConflictingValues()` is a log reading test.

Reply:
On balance I prefer to keep a mix to reduce the boilerplate and reduce the risk of updating one place without updating the other when adding new tests to the servlet or refactoring, but it's not a big deal.

However, putting both tests in one method like this means you'll only get one result for both methods rather  than separate failures for each. If you're going to move them to the class like this, please keep them in separate methods.

Please also update the test servlet with a comment to point out that new tests added to the servlet also need to be added to the test class since that's not the norm.

Reply:
Since I'm changing them anyway and you asked I'll put it back to a mixed paradigm.